docs: pg_search extension guide (#2957)
neondatabase/website: content/docs/extensions/pg-extensions.md

| [pgtap](https://pgtap.org/documentation.html)                                                    |  1.3.3 |  1.3.3 |  1.3.3 |  1.3.3 |                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
| [pgvector](/docs/extensions/pgvector)                                                            |  0.8.0 |  0.8.0 |  0.8.0 |  0.8.0 | Install with `CREATE EXTENSION vector;`                                                                                                                                                                                                                                                                                                                                                                                                                       |

> | [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 15.2  | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer: 0.15.5 should be the new default

Reviewer 2: Fixed.

> | [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 15.5  | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer 2: ```suggestion
| [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 0.15.7  | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

> | [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 0.15.7 | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer 3: ```suggestion
| [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 0.15.8 | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

> | [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 0.15.7  | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer 2: ```suggestion
| [pg_search](/docs/extensions/pg_search)                                      | -  | -  | -  | 0.15.7 | Install with `CREATE EXTENSION pg_search;` on Postgres 17.                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
